https://issues.redhat.com/browse/ACM-14857 (#7235)
open-cluster-management/rhacm-docs: clusters/hosted_control_planes/bm_intro.adoc

* You need to xref:../hosted_control_planes/install_hcp_cli.adoc#hosted-install-cli[install the hosted control plane command line interface].

> * If your hub cluster has a proxy configuration, ensure that it can reach the hosted cluster API endpoint by adding all hosted cluster API endpoints to the `noProxy` field on the `Proxy` object. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/networking/enable-cluster-wide-proxy#enable-cluster-wide-proxy[Configuring the cluster-wide proxy].

Reviewer: My attempt to reduce some of the words, but just a suggestion.
```suggestion
* For a hub cluster that has a proxy configuration to reach the hosted cluster API endpoint, add all hosted cluster API endpoints to the `noProxy` field on the `Proxy` object. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/networking/enable-cluster-wide-proxy#enable-cluster-wide-proxy[Configuring the cluster-wide proxy].
```
